Arrays/2779MaximumBeautyofanArrayAfterApplyingOperation.py

```
@@ -0,0 +1,62 @@
+class Solution:
+    def maximumBeauty(self, nums: list[int], k: int) -> int:
+        count=1
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)